resources/views/inputs/input-with-icon.blade.php

```diff
@@ -29,7 +29,7 @@
 
             @if ($slot ?? false)
                 <div>
-                    <div class="flex {{ $slotClass ?? 'h-full bg-white' }}">
+                    <div class="flex {{ $slotClass ?? 'h-full' }}">
```

Comment:
when you remove this, it gives this issue in the footer
<img width="436" alt="image" src="https://user-images.githubusercontent.com/35610748/112839398-752c4480-909e-11eb-9ded-dd6b179088d1.png">

Reply:
entry is rounded and the icon has no background aspect anymore to be part of the "input"

Reply:
@ItsANameToo the `bg-white` is now moved to msq